api/graphql/schema.graphql

```diff
@@ -139,7 +139,6 @@ type Community {
   settings: String
   invitePath: String
   network: Network
-  numMembers: Int
```

Comment:
looking at the database it seem that num_members is being set and is correct, so why not just use that?